common_arm64.h

```diff
@@ -80,13 +91,22 @@ static void __inline blas_lock(volatile BLASULONG *address){
 
 #if !defined(OS_DARWIN) && !defined (OS_ANDROID)
 static __inline BLASULONG rpcc(void){
+  #ifdef C_MSVC
```

Comment:
Won't this also work on GCC or Clang's GCC frontend?

Reply:
As far as I know, this would be a problem if using MSC_VER in the ifdef here, but not C_MSVC

Reply:
I mean to say that `_ReadStatusReg()` could be available in other compilers too. But I see, the PR as is the most conversative in changes. It was just a suggestion ;-)

Reply:
Right, most conservative change to accomodate the least recommendable (as it cannot use the existing optimized assembly kernels) compiler. Thanks for your suggestion though :)